### PR TITLE
Lm antenna

### PR DESCRIPTION
**Pull request by n7275**
LM steerable antenna tracking now works...as far as I can tell.

The implementation is very similar to the CSM HGA.

### CODE REVIEW

File: Orbitersdk/samples/ProjectApollo/src_lm/lm_telecom.cpp [hunk 1]
```diff
@@ -2470,6 +2470,8 @@ LEM_SteerableAnt::LEM_SteerableAnt()
 	hpbw_factor = 0.0;
 	SignalStrength = 0.0;
 
+	double angdiff = 1.0*RAD;
```
Comment (indy91) [inline Orbitersdk/samples/ProjectApollo/src_lm/lm_telecom.cpp:2473]: This is unused it seems

File: Orbitersdk/samples/ProjectApollo/src_lm/lm_telecom.cpp [hunk 2]
```diff
@@ -2677,6 +2748,42 @@ void LEM_SteerableAnt::DefineAnimations(UINT idx) {
 	lem->SetAnimation(anim_SBandPitch, sband_proc[0]); lem->SetAnimation(anim_SBandYaw, sband_proc[1]);
 }
 
+VECTOR3 LEM_SteerableAnt::pitchYaw2GlobalVector(double pitch, double yaw, MATRIX3 NBSA)
```
Comment (indy91) [inline Orbitersdk/samples/ProjectApollo/src_lm/lm_telecom.cpp:2751]: NBSA is a geometrical constant for this class, doesn't need to be a parameter for this function

File: Orbitersdk/samples/ProjectApollo/src_lm/lm_telecom.cpp [hunk 3]
```diff
@@ -2585,35 +2625,40 @@ void LEM_SteerableAnt::Timestep(double simdt){
...
-	}
-	else
-	{
-		SignalStrength = 0.0;
-	}
+	double LEM_SteerableAntGain = pow(16.5 / 10, 10);
+
+	EarthSignalDist = length(pos - R_E) - oapiGetSize(hEarth); //distance from earth's surface in meters
+
+	LEM_SteerableAntFrequency = 2119; //MHz. Should this get set somewhere else?
```
Comment (indy91) [inline Orbitersdk/samples/ProjectApollo/src_lm/lm_telecom.cpp:2653]: Lots of constants here, maybe shouldn't be done on every timestep?

File: Orbitersdk/samples/ProjectApollo/src_lm/lm_telecom.cpp (hunk 1 above)
Comment (n7275) [reply Orbitersdk/samples/ProjectApollo/src_lm/lm_telecom.cpp:2473]: Removed.

File: Orbitersdk/samples/ProjectApollo/src_lm/lm_telecom.cpp (hunk 2 above)
Comment (n7275) [reply Orbitersdk/samples/ProjectApollo/src_lm/lm_telecom.cpp:2751]: Removed NBSA parameter from function input

File: Orbitersdk/samples/ProjectApollo/src_lm/lm_telecom.cpp (hunk 3 above)
Comment (n7275) [reply Orbitersdk/samples/ProjectApollo/src_lm/lm_telecom.cpp:2653]: yeah, that's probably wasting a bit of time.
Removed.

**Pull request merged**